trikde/triangleplot.py

```diff
@@ -8,11 +8,11 @@
 class TrianglePlot(object):
 
     _default_contour_colors = [(colors.cnames['darkslategrey'], colors.cnames['black'], 'k'),
```

Comment:
you can change this locally when you create the class by setting a new private attribute for _default_contour_colors, no need to change the default setting here